src/runtime/core.ts

```diff
@@ -11,6 +11,20 @@ import { plugin as pythonPlugin } from "starboard-python/dist/index.js";
 import { InboundNotebookMessage } from "../types/messages";
 import { notebookContentToText } from "../content/serialization";
 
+export function respondToStarboardDOMEvents(runtime: Runtime) {
+  const nb = runtime.dom.notebook;
+  const controls = runtime.controls;
+
+  nb.addEventListener("sb:run_cell", (evt) => controls.runCell(evt.detail.id));
+  nb.addEventListener("sb:insert_cell", (evt) => controls.insertCell(evt.detail.id, evt.detail));
+  nb.addEventListener("sb:remove_cell", (evt) => controls.removeCell(evt.detail.id));
+  nb.addEventListener("sb:change_cell_type", (evt) => controls.changeCellType(evt.detail.id, evt.detail));
+  nb.addEventListener("sb:reset_cell", (evt) => controls.resetCell(evt.detail.id));
+  nb.addEventListener("sb:focus_cell", (evt) => controls.focusCell(evt.detail.id, evt.detail));
```

Comment:
I'm considering changing every `controls` function to take a single argument with all the fields, then here they would all be `controls.<functionname>(event.detail)`.

In other words:
Instead of `runtime.controls.changeCellType("my-cell-id", {newCellType: "mytype"})`  it would be 
Instead of `runtime.controls.changeCellType({id: "my-cell-id", newCellType: "mytype"})`